### PR TITLE
[AMORO-1878] Paimon support Optimizing info

### LINKED ISSUE #1878: [Subtask]: Support display paimon optimizing information in optimizing page
### Search before asking

- [X] I have searched in the [issues](https://github.com/NetEase/arctic/issues?q=is%3Aissue) and found no similar issues.

### Description

_No response_

### Parent issue

#1269 

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by HuangFru**
## Why are the changes needed?

Close #1878.

## Brief change log

- Add Paimon's way to get compaction info in TableContriller.
- Implement Paimon's way to get compaction info in ServerTableDescriptor.

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (**yes**)
- If yes, how is the feature documented? (**not applicable**)

### CODE REVIEW

**Comment by HuangFru** [comment]
This is the screenshot:
image

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java [hunk 1]
```diff
@@ -19,8 +19,10 @@
 package com.netease.arctic.server.dashboard.controller;
 
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
```
Comment (baiyangtx) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java:21]: There is a guava dependency,  please fix it with iceberg guava shade instead.

Comment (HuangFru) [reply ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java]: done

**Comment by shidayang** [comment]
> This is the screenshot: image

Standardizing the presentation format of null values.

**Comment by HuangFru** [comment]
> > This is the screenshot: image
> 
> Standardizing the presentation format of null values.

image
Updated.

**Review by shidayang — APPROVED** [review]

**Pull request merged**